src/cf-forms/usage.md

```diff
@@ -12,19 +12,21 @@ The cf-forms component provides extensions to the basic form styles for Capital
 
 - [Variables](#variables)
     - [Color variables](#color-variables)
-    - [Sizing variables](#sizing-variables)
 - [Labels](#labels)
-    - [Label header](#label-header)
-- [Basic Form Elements](#basic-form-elements)
+    - [Basic label](#basic-label)
+    - [Label header](#label-heading)
 - [Inputs](#inputs)
+    - [Input states](#basic-text-inputss)
```

Comment:
Should be `[Basic text inputs](#basic-text-inputs)` (note the single `s` at the end of the link, too)